app/assets/stylesheets/diaries.scss

```diff
@@ -0,0 +1,6 @@
+.single_diary {
+  box-shadow: 1px 3px 1px #d7e4ed;
```

Comment:
Color literals like `#d7e4ed` should only be used in variable declarations; they should be referred to via variable everywhere else.